src/js/mixins/TabsMixin.js

```diff
@@ -96,25 +98,33 @@ const TabsMixin = {
   tabs_getRoutedItem(customClasses, tab) {
     let notificationCount = NotificationStore.getNotificationCount(tab);
     let tabLabelClass = classNames(customClasses, {'tab-item-label': true});
-    let link = (
-      <Link
-        to={tab}
-        className={tabLabelClass}
-        onClick={this.tabs_handleTabClick.bind(this, tab)}>
-        {this.tabs_tabs[tab]}
-      </Link>
-    );
 
     if (notificationCount > 0) {
       return (
-        <span className="badge-container badge-primary">
-          {link}
-          <span className="badge text-align-center">{notificationCount}</span>
-        </span>
+        <Link
+          to={tab}
+          className={tabLabelClass}
+          onClick={this.tabs_handleTabClick.bind(this, tab)}>
+          <span className="tab-label-text">
```

Comment:
I think this classname should actually be `tab-item-label-text` to follow the parent->children relationship (e.g. the parent is `tab-item`, then `tab-item-label`, and here `tab-item-label-text`)